octohatrack/helpers.py

```diff
@@ -7,25 +7,29 @@
 from .generatehtml import generate_html
 from functools import wraps
 
-# Always run on start import
-cache_file = "cache_file.json"
-cache = {}
+print(sys.argv)
```

Comment:
I'll happily use this for my own debugging, but perhaps you'll want the print to eventually be optional?